inference.py

```diff
@@ -296,14 +296,35 @@ def process_image(
     Generate a list of patch features from an image
     """
     if task_description["domain"] == "pathology":
-        max_number_of_tiles = 14_000 if task_description["task_type"] in ["classification", "regression"] else None
+        task_type = task_description["task_type"]
+        max_number_of_tiles = 14_000 if task_type in ["classification", "regression"] else None
+
+        if task_type in ["detection", "segmentation"]:
+            tiling_params = TilingParams(
+                spacing=0.5, tolerance=0.07, tile_size=224, overlap=0,
```

Comment:
Please keep consistent
```suggestion
                spacing=0.5, tolerance=0.07, tile_size=224, overlap=0.0,
```